app/seeders/demo_data/project_seeder.rb

```diff
@@ -134,7 +134,7 @@ def set_project_status(project, key)
 
     def set_members(project)
       role = Role.find_by(name: translate_with_base_url(:default_role_project_admin))
-      user = User.admin.first
+      user = User.user.admin.first
```

Comment:
Why is that change necessary? Shouldn't single table inheritance just filter exactly that?

Reply:
```
pry(main)> User.admin.to_sql
=> "SELECT \"users\".* FROM \"users\" WHERE \"users\".\"type\" IN ('User', 'SystemUser') AND \"users\".\"admin\" = TRUE"
[6] pry(main)> User.user.admin.to_sql
=> "SELECT \"users\".* FROM \"users\" WHERE \"users\".\"type\" IN ('User', 'SystemUser') AND \"users\".\"type\" = 'User' AND \"users\".\"admin\" = TRUE"
```

In my test runs, the SystemUser was assigned sometimes, based on the whim of the database as to what is first.

Reply:
Gotcha.